commit 07ba3ae5e7338e53c08891af874e386b3282765f (refs/changes/98/2498/3)
Author: Brittany Wald <brittany.wald@joyent.com>
Date:   2017-09-06T21:55:26+00:00 (2 years, 1 month ago)
    
    MANTA-3387 node-fash npm publish cruft
    Reviewed by: Kody A Kantor <kody.kantor@gmail.com>
    Approved by: Kody A Kantor <kody.kantor@gmail.com>

diff --git a/.npmignore b/.npmignore
index a56a7ef..3025614 100644
--- a/.npmignore
+++ b/.npmignore
@@ -1,2 +1,6 @@
+deps
 node_modules
-
+test
+tools
+.gitmodules
+Makefile
